servers/src/main/java/tachyon/worker/block/TieredBlockStore.java

```diff
@@ -413,6 +423,19 @@ private void removeBlockNoLock(long userId, long blockId) throws IOException {
     mMetaManager.removeBlockMeta(blockMeta);
   }
 
+  private void updateLockedBlocks() {
```

Comment:
I don't think you need this method and `mLockedBlocks`, and you can simply call `mLockManager.getLockedBlocks` inside `getUpdatedView`